hotkeys.js

```diff
@@ -15,6 +15,7 @@ _.extend(Hotkeys.prototype, {
     add: function(obj) {
         var combo = obj.combo;
         var description = obj.description;
+        var reference = obj.reference; 
```

Comment:
would suggest to name it like described in the mousetrap docs: eventType - what do you think?

> There is a third argument you can use to specify the **type of event** to listen for. It can be keypress, keydown or keyup.